spring-cloud-dataflow-composed-task-runner/src/test/java/org/springframework/cloud/dataflow/composedtaskrunner/properties/ComposedTaskPropertiesTests.java

```diff
@@ -127,12 +127,12 @@ public void testComposedTaskAppArguments() {
 	@Test
 	public void testAssignmentOfOauth2ClientCredentialsClientAuthenticationMethod(){
 		this.contextRunner
-				.withSystemProperties("OAUTH2_CLIENT_CREDENTIALS_CLIENT_AUTHENTICATION_METHOD=POST")
+				.withSystemProperties("OAUTH2_CLIENT_CREDENTIALS_CLIENT_AUTHENTICATION_METHOD=client_secret_post")
```

Comment:
Just a small question - why don't you use
```java
.withSystemProperties("OAUTH2_CLIENT_CREDENTIALS_CLIENT_AUTHENTICATION_METHOD="+ClientAuthenticationMethod.CLIENT_SECRET_POST.value())
```
so that the value of the property is also derived from the enum?

Reply:
@klopfdreh Good point.  There will be a PR coming up soon updating the security in this branch!   Stay tuned!

Reply:
https://github.com/spring-cloud/spring-cloud-dataflow/issues/5883